charts/cassandra/templates/deployment.yaml

```
@@ -155,7 +155,7 @@ spec:
         image: {{ .Values.image }}
         env:
           - name: CASSANDRA_SEEDS
-            value: "cassandra-0.cassandra.default.svc.cluster.local"
+            value: "cassandra-0.cassandra.sample-apps.svc.cluster.local"
```

I am curious if we can make this namespace agnostic by trying
```suggestion
            value: "cassandra-0"
```

---

This did not work.